kafka-ui-react-app/src/components/Topics/Topic/Edit/__test__/topicParamsTransformer.spec.ts

```diff
@@ -0,0 +1,598 @@
+import topicParamsTransformer from 'components/Topics/Topic/Edit/topicParamsTransformer';
+import { TopicWithDetailedInfo } from 'redux/interfaces';
+import { DEFAULTS } from 'components/Topics/Topic/Edit/Edit';
+import { CleanUpPolicy, ConfigSource } from 'generated-sources';
+
+const startParams: TopicWithDetailedInfo | undefined = {
```

Comment:
Just create a ./fixtures file somewhere near this test file to separate such huge objects from the tests.

Reply:
```suggestion
const startParams: TopicWithDetailedInfo = {
```